.golangci.yml

```diff
@@ -125,6 +126,12 @@ issues:
     - text: "SA1019: (assert|cmp|is)\\.ErrorType is deprecated"
       linters:
         - staticcheck
+    - path: api/types
+      linters:
+        - gofumpt
```

Comment:
There was a bit of a back-and-forth on this one. I'm personally a big fan of `gofumpt`, but there were some concerns about enforcing it, because it may not be what contributors have installed, and enabling it as default formatter may be a bit of a PITA if you contribute to multiple projects (some may be using `gofumpt`, others may not, which could cause an over-enthusiastic IDE to reformat unrelated files). If we _do_ start enforcing it, perhaps we need a `make format` (e.g.) target that can run `gofumpt` containerised (using a `Dockerfile` and `docker build --output` or similar to write the results).

- https://github.com/moby/moby/pull/43169#issuecomment-1018324487

(Perhaps opinions have changed though)